examples/integration-tutorials/exposing-several-services-as-a-single-service/README.md

```diff
@@ -1 +1,30 @@
-# exposing-several-services-as-a-single-service
+# Service Chaining
+
+In this example, we're going to integrate many service calls and expose a single service.
+
+## Backend  
+
+We will use the same hospital service as the backend for this example. The hospital service exposes multiple services i.e making an appointment reservation, viewing appointment details, viewing the doctors' details, etc. We will combine two of such services, making an appointment reservation and settling the payment for the reservation, and expose them as a single service in ballerina.  
+
+## Developement
+
+In this example, we will expose a service in ballerina named 'makeReservation'. A client will call this service with the request payload.  This service will first call the reservation service of the backend and obtain a response. Using that response, it will call the payment settlement service of the backend and obtain a response. Finally it will send the response back to the client.
+
```

Comment:
Let's walk through the code segments and explain what it does as well

Reply:
done